networkit/components.pyx

```diff
@@ -140,14 +141,43 @@ cdef class ParallelConnectedComponents(Algorithm):
 		self._this = new _ParallelConnectedComponents(G._this, coarsening)
 
 	def getPartition(self):
+		""" Get a Partition that represents the components.
+
+		Returns:
+		--------
+		networkit.Partition
+			A partition representing the found components.
+		"""
 		return Partition().setThis((<_ParallelConnectedComponents*>(self._this)).getPartition())
 
 	def numberOfComponents(self):
+		""" Get the number of connected components.
+
+		Returns:
+		--------
+		count:
+			The number of connected components.
+		"""
 		return (<_ParallelConnectedComponents*>(self._this)).numberOfComponents()
 
 	def componentOfNode(self, v):
+		"""  Get the the component in which node `v` is situated.
+
+		v : node
+			The node whose component is asked for.
+		"""
 		return (<_ParallelConnectedComponents*>(self._this)).componentOfNode(v)
 
+	def getComponents(self):
+		""" Get the connected components, each as a list of nodes.
+
+		Returns:
+		--------
+		list:
+			The connected components.
+		"""
+		return (<_ConnectedComponents*>(self._this)).getComponents()
```

Comment:
Shouldn't you cast to `ParallelConnectedComponents`?

Reply:
Yes ;)